geode-connectors/src/main/java/org/apache/geode/connectors/jdbc/internal/cli/CreateDataSourceCommand.java

```diff
@@ -138,7 +138,11 @@ public ResultModel createJDNIBinding(
       result.setConfigObject(configuration);
       return result;
     } else {
-      return ResultModel.createInfo("No members found.");
+      if (service != null) {
```

Comment:
What's the difference between the two cases when service is not null and service is null? The error messages are the same. However one is a info, and the other is an error. I assume the messages would be different. Would you please explain a bit more?

Reply:
The error message should be: No members found and cluster configuration disabled.
The info message is just telling you that no members were found but it was still created in cluster config.